From b17a9b9cc9438b6dfac4402ac4107f08e23a4373 Mon Sep 17 00:00:00 2001
From: Chris Michael <cp.michael@samsung.com>
Date: Wed, 29 May 2013 11:50:17 +0000
Subject: Backport: 45b4d9f :: Move xinerama init until After randr has setup the screens.

NB: Old Man, THIS should fix your black screen problem ;)

Signed-off-by: Chris Michael <cp.michael@samsung.com>
Signed-off-by: Deon Thomas <PrinceAMD.Elive@gmail.com>
---
diff --git a/src/bin/e_main.c b/src/bin/e_main.c
index ee8fd8a..ed58cfd 100644
--- src/bin/e_main.c
+++ src/bin/e_main.c
@@ -462,15 +462,6 @@ main(int argc, char **argv)
    TS("E_Alert Init Done");
    _e_main_shutdown_push(e_alert_shutdown);
 
-   TS("E_Xinerama Init");
-   if (!e_xinerama_init())
-     {
-        e_error_message_show(_("Enlightenment cannot initialize E_Xinerama!\n"));
-        _e_main_shutdown(-1);
-     }
-   TS("E_Xinerama Init Done");
-   _e_main_shutdown_push(e_xinerama_shutdown);
-
    TS("E_Hints Init");
    e_hints_init();
    TS("E_Hints Init Done");
@@ -521,6 +512,15 @@ main(int argc, char **argv)
      _e_main_shutdown_push(e_randr_shutdown);
    TS("E_Randr Init Done");
 
+   TS("E_Xinerama Init");
+   if (!e_xinerama_init())
+     {
+        e_error_message_show(_("Enlightenment cannot initialize E_Xinerama!\n"));
+        _e_main_shutdown(-1);
+     }
+   TS("E_Xinerama Init Done");
+   _e_main_shutdown_push(e_xinerama_shutdown);
+
    TS("E_Env Init");
    if (!e_env_init())
      {
--
cgit v0.9.2
